app/assets/javascripts/update_inline.js

```diff
@@ -0,0 +1,46 @@
+function updateInline() {
+  $('.ideas').delegate('.title', 'click', function() {
+    var $idea = $(this).closest(".idea");
+    document.getElementById("ideaTitle" + $idea.attr('data-id')).contentEditable = true;
+    $("#ideaTitle" + $idea.attr('data-id')).keyup(function(event){
+      if(event.keyCode === 13){
+        event.preventDefault();
+        inlineUpdateIdea($idea);
+      }
+    });
+  });
+
+  $('.ideas').delegate('.body', 'click', function() {
```

Comment:
It seems like this event and the one above aren't particularly DRY. Could you write one function that could handle both situations?